source/deploying-with-ansible/guide/install-wazuh-server.rst

```diff
@@ -37,17 +37,17 @@ We can see the roles we have.
 	│   │   ├── tasks
 	│   │   └── templates
 	│   ├── ansible-kibana
-	│   │   ├── defaults
-	│   │   ├── handlers
-	│   │   ├── meta
-	│   │   ├── tasks
-	│   │   └── templates
-	│   └── ansible-logstash
-	│       ├── defaults
-	│       ├── handlers
-	│       ├── meta
-	│       ├── tasks
-	│       └── templates
+	│       ├── defaults
+	│       ├── handlers
+	│       ├── meta
+	│       ├── tasks
+	│       └── templates
+	│    
```

Comment:
Please fix this indentation.

Reply:
Hi @manuasir 

Done!

Thanks for paying attention to such details.

Kind regards,

Rshad